release/README.md

```diff
@@ -2,8 +2,37 @@
 
 This directory contains tools for Kata Containers releases.
 
+## update-repository-version.sh ##
+
+This script is used to create a Github pull request (a.k.a PR) to change version in all
+the Kata repositories.
+
+To get more information about how to use it , run:
+
+```bash
+$ ./update-repository-version.sh -h
+```
+
+### Update Kata projects to a new version ###
+Kata Containers is devided in multiples projects. On each release all the
```

Comment:
typo "devided" to "divided" + lines 17-18 suggested rewrite: 
Kata Containers is divided into multiple projects. With each release, all project versions are updated to keep the version consistent. 
or 
Kata Containers is divided into multiple projects. With each release, all versions for all projects are updated to keep the versions consistent.